cmd/podman/shared/funcs.go

```diff
@@ -6,20 +6,22 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/containers/libpod/pkg/util"
 	"github.com/google/shlex"
 )
 
 func GetAuthFile(authfile string) string {
 	if authfile != "" {
 		return authfile
 	}
+
 	authfile = os.Getenv("REGISTRY_AUTH_FILE")
 	if authfile != "" {
 		return authfile
 	}
-	runtimeDir := os.Getenv("XDG_RUNTIME_DIR")
-	if runtimeDir != "" {
-		return filepath.Join(runtimeDir, "containers/auth.json")
+
+	if runtimeDir, err := util.GetRuntimeDir(); err == nil {
```

Comment:
Is it OK to just silently ignore the error here?

Reply:
Yes I think we can.